tests/utils/tools.php

```diff
@@ -686,6 +688,8 @@ function fromJSON($var) {
 
 /* Note: this fail point may terminate the mongod process, so you may want to
  * use this in conjunction with a throwaway server. */
-function failGetMore(MongoDB\Driver\Manager $manager) {
-    return configureFailPoint($manager, "failReceivedGetmore", "alwaysOn");
+function failGetMore(MongoDB\Driver\Manager $manager)
+{
+    configureFailPoint($manager, 'failReceivedGetmore', 'alwaysOn');
+    configureFailPoint($manager, 'failCommand', 'alwaysOn', [ 'errorCode' => 237, 'failCommands' => ['getMore'] ]);
```

Comment:
We use ``237`` here, as that's the same original code that MongoD would throw if a cursor had already gone by the time we call ``getMore``. This allows us to make things consistent with the getMore OP behaviour from previous mongod versions. An ``errorCode`` is required here for the failPoint to work.

Reply:
This seems like something worth adding to a block comment for this line or method.